automatic/7zip.commandline/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,33 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
+
+$filePath32    = "$toolsDir\7zip_x32.exe"
+$filePath64    = "$toolsDir\7zip_x64.exe"
+$filePathExtra = "$toolsDir\7zip_extra.7z"
+
+$packageArgs = @{
+  packageName = '7zip.commandline'
+  destination = "$toolsDir"
+  file = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne $true) {
+    Write-Host "Installing 64 bit version" ; $filePath64
+  } else {
+    Write-Host "Installing 32 bit version" ; $filePath32
+  }
+}
+Get-ChocolateyUnzip @packageArgs
+
+$packageArgs.packageName = '7zip.commandline Extras'
+$packageArgs.destination = "$toolsDir\7z-extra"
+$packageArgs.file        = $filePathExtra
+Get-ChocolateyUnzip @packageArgs
+
+Remove-Item -Path "$toolsDir\Uninstall.exe",$filePath32,$filePath64,$filePathExtra -Force
```

Comment:
Add `-ea 0` to avoid install failing in case this fails, like it was done in other packages